syntax = "proto3";
package tinkoff.proto;

// Request
message AddDeponentToDealStepRequest {
  string deal_id = 1;
  string step_id = 2;
  string beneficiary_id = 3;
  float amount = 4;
}

// Response
message AddDeponentToDealStepResponse {
  string deal_id = 1;
  string step_id = 2;
  string beneficiary_id = 3;
  float amount = 4;
}
